packages/core/src/components/tabs/_tabs.scss

```diff
@@ -40,7 +40,9 @@ $tab-indicator-width: 3px !default;
     flex-direction: column;
 
     .#{$ns}-tab {
+      align-items: center;
       border-radius: $pt-border-radius;
+      display: flex;
```

Comment:
it seems like we are not demonstrating the behavior of this styling change in the docs example